app/ts/components/TransactionPage/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { ConnectAccount } from '../ConnectAccount.js'
+import { RecentTransfers } from '../RecentTransfers.js'
+import { useRouter } from '../HashRouter.js'
+import { TransactionResponseQuery } from './TransactionResponse.js'
+import { useSignal } from '@preact/signals'
+import { TransactionReceiptQuery } from './TransactionReceipt.js'
+import { TransactionResponse } from '../../types.js'
+
+const SCROLL_OPTIONS = { inline: 'start', behavior: 'smooth' } as const
+
+export const TransactionPage = () => {
+	return (
+		<div class='fixed inset-0 bg-black text-white h-[100dvh]'>
+			<Root>
+				<LeftPanel />
+				<MainPanel />
+			</Root>
+		</div>
+	)
+}
+
+const MainPanel = () => {
+	const { nav, main } = usePanels()
+	const router = useRouter<{ transaction_hash: string }>()
+	const transactionResponse = useSignal<TransactionResponse | undefined>(undefined)
+
+	return (
+		<Main>
+			<Header>
+				<HeaderNav show={main?.isIntersecting} iconLeft={MenuIcon} onClick={() => nav?.target.scrollIntoView(SCROLL_OPTIONS)} text='Menu' />
+			</Header>
+
+			<div class='px-4'>
+				<ConnectAccount />
+			</div>
+
+			<div class='px-4'>
+				<div class='py-4'>
+					<div class='text-3xl font-bold'>Transaction Details</div>
+				</div>
+			</div>
+
+			<div class='px-4'>
+				<div class='grid gap-2'>
+					<TransactionResponseQuery transactionHash={router.value.params.transaction_hash} onSuccess={response => transactionResponse.value = response} />
+					<TransactionReceiptQuery transactionHash={router.value.params.transaction_hash} transactionResponse={transactionResponse.value} />
+				</div>
+			</div>
+		</Main>
+	)
+}
+
+const LeftPanel = () => {
+	const { nav, main } = usePanels()
+
+	return (
+		<Navigation>
+			<Header>
+				<HeaderNav show={nav?.isIntersecting} iconLeft={CloseIcon} onClick={() => main?.target.scrollIntoView(SCROLL_OPTIONS)} text='Close Menu' />
+			</Header>
+
+			<div class='mb-4 p-4'>
+				<div class='flex items-center gap-2'>
+					<img class='w-10 h-10' src='/img/icon-lunaria.svg' />
+					<div>
+						<div class='text-3xl font-bold leading-none'>Lunaria</div>
+						<div class='text-white/50 leading-none'>Decentralized Wallet</div>
+					</div>
+				</div>
+			</div>
+
+			<div class='pl-4 mb-4'>
+				<div class='text-white/30 text-sm'>Actions</div>
+				<a href="/">
+					<div class='grid grid-cols-[auto,1fr] items-center gap-4 mb-4'>
+						<div class='bg-white/30 w-10 h-10 rounded-full' />
+						<div class='py-2 leading-tight'>
+							<div class='font-bold'>New Transfer</div>
+							<div class='text-white/50'>Send and Manage Tokens</div>
+						</div>
+					</div>
+				</a>
+			</div>
+
+			<RecentTransfers />
+
+			<div class='pl-4 mb-4'>
+				<div class='text-white/30 text-sm mb-2'>Support</div>
+				<div class='p-4 text-sm bg-white/5 border border-white/10 text-white/50'>
+					Join our discord channel to get support from our active community members and stay up-to-date with the latest news, events, and announcements.
```

Comment:
add link

Reply:
i was going to move it to another PR but would just be a small tweak so committed it here [0092596](https://github.com/DarkFlorist/lunaria/pull/115/commits/0092596661526506ef64efc06297c927655576df)